App/arca/assets/css/enterPin.scss

```diff
@@ -0,0 +1,141 @@
+$main-color: #483D8B;
```

Comment:
you could create a sass file to keep all. colors in the same place and then import it in each sass file